pkg/api/store/cluster/cluster_store.go

```diff
@@ -473,7 +473,7 @@ func (r *Store) Update(apiContext *types.APIContext, schema *types.Schema, data
 		return nil, err
 	}
 
-	if err := validateNetworkFlag(data, false); err != nil {
+	if err := validateNetworkFlag(data, existingCluster, false); err != nil {
```

Comment:
we already pass create=false here, we could use it?

Reply:
I wanted to keep it as much the same as possible, the line that I changed in that function is where it gets applied and that is different than the bool. If we want to use the bool, I need to rewrite the function.

Reply:
don't understand, I think the create bool is for the same usage, to put values only if it's create

Reply:
Thanks, I updated it to use the bool